docs/en/quick-answers/troubleshooting/context-missing-task-step.md

```diff
@@ -1,12 +1,31 @@
 ---
 layout: default
-title: My context is missing the Task/Step but it exists as part of the filename.
+title: Why is my context missing the Task/Step when it exists as part of the filename?
 pagename: context-missing-task-step
 lang: en
 ---
 
-# My context is missing the Task/Step but it exists as part of the filename.
+# Why is my context missing the Task/Step when it exists as part of the filename?
 
-Toolkit uses the path cache to help associate paths on disk with entities in Shotgun and build your context. Since the path cache only stores **folder** information and not filenames, any context-related information contained in the filename (including Task or Step name) will not be detected when building the context.
+When you create folders via Toolkit it registers the path against the entity so that a lookup may be performed. This means that given a path, you can determine the correct context.
+Toolkit will only create registries for folders generated from the schema, and so it doesn't consider things like file names or folders that were defined solely in the `templates.yml`.
+If you don't have a `Task` folder in your schema then you can get in a situation where Toolkit needs to know the task of the file but is unable to work out the task from the path alone.
 
-The best way to solve this currently, is to consider modifying your schema slightly so that the Task or Step is part of the folder structure rather than the filename. 
\ No newline at end of file
+**Example**
+
+Take the default schema structure below; `Asset` and `Step` folders will be registered during the folder creation process:
+
+![Default Asset schema](./images/asset-schema.png)
+
+If you generated a file path using a template like this: `assets/{sg_asset_type}/{Asset}/{Step}/work/maya/{task_name}_{name}.v{version}.{maya_extension}'`, and then attempted to figure out the context from that generated path, it would only be able to establish the `Asset` and the `Step` and **not** the `Task` despite the task's name being in the file path.
```

Comment:
Should we put the path on its own line? It's really long and wrapping across lines, so it might be easier to read that way. 

Also that path has an errant single quote at the end.

Reply:
Done good idea!